### PR TITLE
Add service providers that come from the user configuration

### PR DESCRIPTION
**Pull request by ethervoid**

### CODE REVIEW

File: server/extension/cdb_dataservices_server--0.13.2--0.13.1.sql [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+  return isolines
+$$ LANGUAGE plpythonu;
+
+CREATE OR REPLACE FUNCTION cdb_dataservices_server.cdb_mapzen_isochrone(username TEXT, orgname TEXT, source geometry(Geometry, 4326), mode TEXT, range integer[], options text[] DEFAULT array[]::text[])
+RETURNS SETOF cdb_dataservices_server.isoline AS $$
+  plpy.execute("SELECT cdb_dataservices_server._connect_to_redis('{0}')".format(username))
+  redis_conn = GD["redis_connection_{0}".format(username)]['redis_metrics_connection']
+  plpy.execute("SELECT cdb_dataservices_server._get_isolines_config({0}, {1})".format(plpy.quote_nullable(username), plpy.quote_nullable(orgname)))
+  user_isolines_config = GD["user_isolines_routing_config_{0}".format(username)]
```
Comment (iriberri) [inline server/extension/cdb_dataservices_server--0.13.2--0.13.1.sql]: I think this is wrong. The old version had the specific mapzen keys in the dictionary: https://github.com/CartoDB/dataservices-api/blob/91c715f9f560783ed3081e228b99a5ec42b0fc73/server/extension/cdb_dataservices_server--0.13.1.sql#L2272

File: server/lib/python/cartodb_services/cartodb_services/metrics/config.py [hunk 2]
```diff
@@ -104,17 +104,26 @@ def service_type(self):
...
+        if not self._routing_provider:
+            self._routing_provider = self.DEFAULT_PROVIDER
         self._mapzen_api_key = self._db_config.mapzen_routing_api_key
         self._monthly_quota = self._db_config.mapzen_routing_monthly_quota
         self._period_end_date = date_parse(self._redis_config[self.PERIOD_END_DATE])
 
     @property
     def service_type(self):
-        return 'routing_mapzen'
+        if self._routing_provider == self.MAPZEN_PROVIDER:
```
Comment (iriberri) [inline server/lib/python/cartodb_services/cartodb_services/metrics/config.py]: :D

**Comment by iriberri** [comment]
:+1:

**Comment by ethervoid** [comment]
retest this please

**Pull request merged**